static/app/views/issueDetails/streamline/groupActivityIcons.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  IconAdd,
+  IconCheckmark,
+  IconClose,
+  IconDelete,
+  IconEdit,
+  IconFile,
+  IconFire,
+  IconFlag,
+  IconGraph,
+  IconLock,
+  IconMute,
+  IconNext,
+  IconPlay,
+  IconPrevious,
+  IconRefresh,
+  IconUnsubscribed,
+  IconUser,
+} from 'sentry/icons';
+import {GroupActivityType} from 'sentry/types/group';
+
+export const GroupActivityTypeIconMapping: Record<GroupActivityType, React.ReactNode> = {
+  [GroupActivityType.NOTE]: <IconFile />,
```

Comment:
i think you want to do just IconFile and render it later since this version makes it hard to pass any props to and renders them when the app loads. Here's one with icons i tried recently https://github.com/getsentry/sentry/blob/e7cc9c4a53a6a872b7a10a029917fb5b9908c9ec/static/app/components/events/errorLevelText.tsx#L23-L30

```suggestion
  [GroupActivityType.NOTE]: IconFile,
```